Add Spell Effects to Circle of Protection

avagdumortesin opened the pull request:
Fixes https://github.com/foundryvtt/pf2e/issues/2307

SpartanCPA commented on packs/data/spells.db/protection.json:
```diff
@@ -26,14 +27,19 @@
         "hasCounteractCheck": {
             "value": false
         },
+        "heightening": {
+            "damage": {},
+            "interval": 1,
+            "type": "interval"
+        },
```
Why?

avagdumortesin replied:
On Protection, all I edited was the description text to add the `<hr />` tag and change the paragraphs that had the Spell Effects into an unordered list. This change happened from opening the spell in this version of the system.

avagdumortesin replied:
New commit removes that; I'm not certain why it was added, because I did not click on either of the Heightening buttons on the spell.

SpartanCPA commented:
I'm ok with the actual changes to Protection. You need to clean up the files tab though, as there are a *lot* of superflous changes. Maybe rebase?

avagdumortesin commented:
I had rebased on `master` prior to starting work on it again last night, and it wound up pulling all the commits in, which caused all the other files to be added.

avagdumortesin commented:
There we go; now showing only 18 files changed:

- Removed the 4 previous Protection Spell Effects
- Added new Protection Spell Effect
- Added the link to the new spell effect to the Protection and Circle of Protection spells
- Replaced the 4 previous spell effects with the new spell effect on 11 NPCs in various compendiums that had Protection as a spell
